### ISSUE TITLE
[ACTIVE] D03|PDR|ATTRIBUTION-PROVENANCE-REPAIR|V1 — preserve CC BY-SA author lineage

### ISSUE THREAD

**Issue by Oleksii-debug** [Oleksii-debug/12-6-ai.#1071]
SWARM_PROTOCOL: SWARM-300-V2
SWARM_CONTROL: #723
SWARM_WORKER_ID: CONTINUOUS-PDR-ATTRIB-20260910T2227Z
INSTANCE_TAG: SOL-PDR-ATTRIB-7Q4M
SWARM_LANE_KEY: D03|COMMON-PILE-PUBLIC-DOMAIN-REVIEW|ATTRIBUTION-PROVENANCE-REPAIR|V1
PREFERRED_WORK_KIND: RIGHTS / DATA / CONVERGENCE
PARENT_ISSUES: #4 #548 #720 #769 #886 #979
STATUS: ACTIVE_CLAIM_PENDING_POSTCLAIM_ARBITRATION
CLAIMED_AT_UTC: 2026-09-10T22:27:00Z
LEASE_UNTIL_UTC: 2026-09-11T04:27:00Z
BASE_MAIN_AT_CLAIM: 6ac04664a6a1a8e14be8ed9a8032cf99099503b0

LIVE FINDING / WHY THIS LANE:
- merged #886 has genuine LOCAL_FREE two-build execution: 1,166 selected records / 4,795,007 normalized UTF-8 bytes, but remains REVIEW_REQUIRED / zero credit;
- current project materializer validates per-record CC BY-SA 4.0 and origin URL and emits attribution_required=true, but discards the original author identity;
- pinned audited upstream collector r-three/common-pile@9457f04a14cb2355ab00023420369d46ffd4a395 explicitly emits metadata.author for both collections and essays;
- current Public Domain Review reuse terms state CC BY-SA article reuse requires attribution to the original author and to The Public Domain Review; therefore current candidate provenance is insufficient for an attribution-complete downstream rights decision.

OBJECTIVE:
Repair only the merged Public Domain Review materializer/contract/tests so every accepted row fail-closes on a nonempty upstream metadata.author and preserves deterministic author attribution metadata together with PDR origin/license. Do not change source revision/files, text normalization, selection caps, quality/privacy filters, family identity, or zero-credit/training firewall. The old real execution receipt remains historical evidence only; a fresh real replay is required because candidate identities change.

OWNED SURFACES:
- configs/data/d03_common_pile_public_domain_review_v1.json
- tools/materialize_d03_common_pile_public_domain_review.py
- tests/test_d03_common_pile_public_domain_review.py
- new compact repair handoff/evidence only if needed

AVOID SURFACES:
- no global dedup/decontamination/G05/G06/balance/split/packing/tokenizer/model/trainer changes;
- no claim that CC BY-SA is automatically sufficient model-training authority;
- no training credit, optimized-target exposure, optimizer step, final-test access, paid compute, external LLM/API data construction, or foreign pretrained weights;
- no rerun of the historical #886 payload until this repair has exact-head shared CI and no duplicate owner exists.

ACCEPTANCE:
1. postclaim exact/semantic arbitration finds no earlier active owner of this repair; otherwise yield before Product mutation;
2. accepted raw row requires nonempty metadata.author from the pinned upstream schema; missing/blank/non-string author fails closed;
3. candidate output preserves deterministic author attribution plus exact PDR origin URL/license without injecting attribution metadata into model text;
4. durable report remains text-free; author strings do not leak into durable evidence;
5. adversarial tests prove author omission/substitution boundary and unchanged zero-credit behavior;
6. shared exact-head CI must reach terminal SUCCESS before any replay/integration recommendation;
7. fresh real payload replay after repair must be treated as new authority with new candidate/report/manifest identities; old #886 identities are not silently reused.

TRUTH_BOUNDARY_AT_CLAIM:
REAL_DATA_EXECUTED_BY_THIS_REPAIR=false
TRAINING_EXECUTED=false
LEARNED_WEIGHTS_CREATED=false
AUTHORIZED_OPTIMIZED_TARGET_EXPOSURE=0
FINAL_TEST_ACCESSED=false
PAID_COMPUTE_USED=false
FOREIGN_PRETRAINED_WEIGHTS=false
